### DISCUSSION TITLE
self-contained: true including audio files

### DISCUSSION THREAD

**Discussion by harrelfe** [quarto-dev/quarto-cli#624] [Q&A] [answered]
`self-contained: true` seems to be going too far in one application.  I wish it to cause generated `png` graphics to be included inside the `.html` file, but it's also causing audio files from the web to be included, making the `html` file 70MB in size.  I link to audio files using the following:

    <audio controls src="https://.....mp3"></audio>

Is there a way to make `quarto` leave the `.mp3` files as links that are streamed/downloaded on demand?  I guess that the usual URL link approach will do that but it doesn't give you the audio player that `<audio ...>` generates.

**Comment by cderv** [comment, marked as answer]
Hi @harrelfe, 

Quarto should respect Pandoc feature - Pandoc will embeded any linked media file, unless the `data-external="1"` attributes is set. 
See in https://pandoc.org/MANUAL.html#option--self-contained

So adding the attributes on your element should make Pandoc ignore those `<audio>` tag.
